test/commons/color/element-is-distinct.js

```diff
@@ -1,271 +1,355 @@
 describe('color.elementIsDistinct', () => {
-  let styleElm;
-  let elementIsDistinct;
+  const elementIsDistinct = axe.commons.color.elementIsDistinct;
+  const fixtureSetup = axe.testUtils.fixtureSetup;
+  const fixture = document.querySelector('#fixture');
+
+  function convertStylePropsToString(props = {}) {
+    return Object.entries(props)
+      .map(([name, value]) => `${name}: ${value}`)
+      .join(';');
+  }
 
-  const fixture = document.getElementById('fixture');
+  function createTestFixture({ target, root, inline } = {}) {
```

Comment:
Did some cleanup of this file. It didn't need to create actual style sheets for the elements so just made a simpler function to create inline styles.